src/interface/src/app/plan/create-scenarios/create-scenarios.component.ts

```diff
@@ -176,6 +177,9 @@ export class CreateScenariosComponent implements OnInit {
           return;
         }
 
+        // Updating breadcrumbs
+        this.planStateService.updateStateWithScenario(scenario.id || null, scenario.name || null)
```

Comment:
do you need  `|| null`? cant you just do  this.planStateService.updateStateWithScenario(scenario.id, scenario.name)

Reply:
Kind of solved with the other comment you did.